### PR TITLE
Support custom lnd connections

### PR DESCRIPTION
**Pull request by mrfelton**
This PR exposes the lnd connection params as environment variables that make it simple to run Zap with an external lnd node.

I've implemented dotenv (https://github.com/motdotla/dotenv) to enable the settings to be controlled by a `.env` file placed at the project root.

Addresses #326 and #324

### CODE REVIEW

**Comment by Empact** [comment]
Also #167

File: app/main.dev.js [hunk 1]
```diff
@@ -257,12 +253,14 @@ app.on('ready', async () => {
 
   sendGrpcDisconnected()
   // Check to see if an LND process is running
-  lookup({ command: 'lnd' }, (err, results) => {
+  lookup({ command: 'node' }, (err, results) => {
```
Comment (Empact) [inline app/main.dev.js]: Why `node` here? The executable is `lnd`, `node` seems overly broad.

Comment (mrfelton) [reply app/main.dev.js]: Good catch @Empact - that was left over from when I was testing some thing locally, before I introduced the `ZAP_LND_DISABLE` flag. Have removed and updated the PR

**Comment by mrfelton** [comment]
Superseded by https://github.com/LN-Zap/zap-desktop/pull/351 so closing this PR.

**Pull request closed without merging**